daemon/api.go

```diff
@@ -845,7 +845,7 @@ func snapUpdateMany(inst *snapInstruction, st *state.State) (msg string, updated
 	case 0:
 		// not really needed but be paranoid
 		if len(inst.Snaps) != 0 {
-			return "", nil, nil, fmt.Errorf("internal error: when asking for a refresh of %s no update was found but no error was generated", strutil.Quoted(inst.Snaps))
+			msg = fmt.Sprintf("Refresh snaps %s: no updates", strutil.Quoted(inst.Snaps))
```

Comment:
Same comment as the other review from last week: listing two or at most three snaps might be okay. More than that and we should instead show something like "Refresh snaps A, B and N more.".